ui/app/components/clients/date-range.hbs

```diff
@@ -3,65 +3,68 @@
-            {{#each this.historicalBillingPeriods as |period idx|}}
-              <D.Checkmark
-                {{on "click" (fn this.updateEnterpriseDateRange period)}}
-                data-test-date-range-billing-start={{add idx 1}}
-                @selected={{eq this.selectedStart (this.formattedDate period)}}
-              >
-                {{this.formattedDate period}}
-              </D.Checkmark>
-            {{/each}}
-          {{/if}}
-        </Hds::Dropdown>
-      {{else}}
+    {{#if this.version.isEnterprise}}
+      <Hds::Text::Display @tag="p" @size="100" class="has-bottom-margin-xs">
+        Change billing period
+      </Hds::Text::Display>
+      <Hds::Dropdown class="has-left-margin-xs" @matchToggleWidth={{true}} as |D|>
+        <D.ToggleButton @text="Billing start date" @color="secondary" data-test-date-range-edit />
+        <D.Description @text="Current period" />
+        <D.Checkmark
+          {{on "click" (fn this.updateEnterpriseDateRange @billingStartTime)}}
+          @selected={{eq this.selectedStart (this.formattedDate @billingStartTime)}}
+          data-test-date-range-billing-start="0"
+        >
+          {{this.formattedDate @billingStartTime}}
+        </D.Checkmark>
+        {{#if this.historicalBillingPeriods.length}}
+          <D.Separator />
+          <D.Description @text="Historical periods" />
+          {{#each this.historicalBillingPeriods as |period idx|}}
+            <D.Checkmark
+              {{on "click" (fn this.updateEnterpriseDateRange period)}}
+              data-test-date-range-billing-start={{add idx 1}}
+              @selected={{eq this.selectedStart (this.formattedDate period)}}
+            >
+              {{this.formattedDate period}}
+            </D.Checkmark>
+          {{/each}}
+        {{/if}}
+      </Hds::Dropdown>
+    {{else}}
+      {{#unless (and this.startDate this.endDate)}}
         <Hds::Button
           class="has-left-margin-xs"
           @text="Set date range"
           @icon="edit"
-          {{on "click" (fn (mut this.showEditModal) true)}}
+          {{on "click" (fn @setEditModalVisible true)}}
           data-test-date-range-edit
         />
-      {{/if}}
-    </div>
+      {{/unless}}
+    {{/if}}
 
   </div>
 
-  {{#if this.showEditModal}}
+  {{#if @showEditModal}}
     <Hds::Modal data-test-date-range-edit-modal @onClose={{this.onClose}} as |M|>
       <M.Header>
         Edit date range
       </M.Header>
       <M.Body>
         <p class="has-bottom-margin-s">
           The start date will be used as the client counting start time and all clients in that month will be considered new.
+          We recommend setting this date to when Vault was deployed to get the most accurate new and total client count
+          estimations. These dates are only for querying data in storage. Only historical data can be queried, so selecting
+          the current month as the end date will not return a value.
```

Comment:
Selecting the current month is prohibited for both the start/end date, right?
```suggestion
          estimations. These dates are only for querying data in storage. Only historical data can be queried, so selecting
          the current month is not available.
```

This text is quite long and most people will probably skim it anyway, so condensing it even more might be ideal.
```suggestion
          estimations. These dates are only for querying data in storage; only historical data can be queried.
```